### DISCUSSION TITLE
AudioCraft Plus v2.0.1

### DISCUSSION THREAD

**Discussion by GrandaddyShmax** [GrandaddyShmax/audiocraft_plus#37] [Announcements]
Change log:

v2.0.1:
- Changed custom model loading to support the official trained models
- Additional changes from the main facebookresearch repo

<hr /><em>This discussion was created from the release <a href='https://github.com/GrandaddyShmax/audiocraft_plus/releases/tag/2.0.1'>AudioCraft Plus v2.0.1</a>.</em>

**Comment by Ariffffff** [comment]
Thankyou, loved that we can move models to inside folder rather than .cache
UI is better than music gen. gave me better results
